class/defaults.yml

```diff
@@ -1,4 +1,43 @@
 parameters:
   etcd:
-    =_metadata: {}
-    namespace: syn-etcd
+    =_metadata:
+      multi_instance: true
+
+    namespace: syn-${_instance}
+
+    secrets: {}
+
+    images:
+      etcd:
+        registry: docker.io
+        repository: bitnami/etcd
+        tag: 3.5.9-debian-11-r76
+      os-shell:
+        registry: docker.io
+        repository: bitnami/os-shell
+        tag: 11-debian-11-r16
+
+    charts:
+      etcd:
+        source: https://charts.bitnami.com/bitnami
+        version: "9.1.0"
+
+    helm_release_name: ${_instance}
+    helm_values:
+      image: ${etcd:images:etcd}
+      volumePermissions:
+        image: ${etcd:images:os-shell}
+      auth:
```

Comment:
Values are updated to remove helm's secret generation and for security by default.